server/index.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
+const bodyparser = require('body-parser');
+const db = require('../database/index.js');
+const Promise = require('bluebird');
```

Comment:
You have modules in here that aren't used. Avoid adding them until you actually need them.